File: leetsync/3830-find-closest-person/find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int dist1 = Math.abs(x - z);
        int dist2 = Math.abs(z - y);
        if (dist1 == dist2) {
            return 0;
        } else if (dist1 < dist2) {
            return 1;
        } else {
            return 2;
        }
    }
}

File: leetsync/3830-find-closest-person/find-closest-person.go
func findClosest(x int, y int, z int) int {
	dist1 := AbsInt(x - z)
	dist2 := AbsInt(z - y)
	if dist1 == dist2 {
		return 0
	} else if dist1 < dist2 {
		return 1
	} else {
		return 2
	}
}
func AbsInt(x int) int {
	if x < 0 {
		return -x
	}
	return x
}